index.md

```diff
@@ -31,12 +38,28 @@ permalink: /
 7. Now you could choose additional modules, but for an easy start, we recommend using the default selection. Click on “Save and Continue” for the last time.
 8. You’re done! Your Thunder site opens and you can start the tour to show you around.
 
-## What is Thunder
-1. Detailed Feature-List
-2. Who is doing it
-3. How can I contact
-4. Slack
-5. Roadmap (only main features)
+## What is Thunder?
+### 1. Detailed Feature-List
+### 2. Who is doing this: The Thunder Core Team
```

Comment:
I think we should keep questions and answers here. So we should use "Who is doing this?"